x-pack/plugins/security_solution/server/endpoint/routes/policy/handlers.test.ts

```diff
@@ -236,6 +246,80 @@ describe('test policy response handler', () => {
       });
     });
   });
+  describe('test GET policy list handler', () => {
```

Comment:
we should add some integration tests as well. I'm guessing in here: https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_endpoint_api_int/apis/policy.ts.